tests/kw_include_test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+include ./tests/utils
+
+function suite
+{
+  suite_addTest "include_twice_Test"
+}
+
+function include_twice_Test()
+{
+  ID=1
+  include ./src/kw_include.sh
+  output=$(include ./src/kw_include.sh)
+  expected_result="255"
+  assertEquals "($ID)" "$expected_result" "$output" 
+}
```

Comment:
I think you can expand this test a little bit more... Some examples:
1. Test the wrong path;
2. Include case that works;
3. Double inclusion;
4. Wrong path parameter;

Finally, squash this change in the first commit.

Reply:
Also, take care of extra white space at the end of the line.

Reply:
What is the difference between 1 and 4?

Reply:
@rodrigosiqueira could you clarify what is the difference between cases 1 and 4?